src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+
+type Props = {
+  setFilter: (filter?: boolean) => void
+};
+
+enum TypeFilter {
+  'all',
```

Comment:
read more about enums https://www.typescriptlang.org/docs/handbook/enums.html#string-enums
and change all types in code to `TypeFilter.ALL`, `TypeFilter.ACTIVE` ...